src/Assignment-3/assignment3.css

```diff
@@ -1,7 +1,96 @@
-/* Our CSS file. */
+body{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-color: gray;
+    padding-top:30px;
+   
+  }
+  
+  .container{
+      height: 400px;
+      width: 600px;
+      border: 1px solid black; 
+      background-color:#f2f2f2;
+      display: flex;
+      align-items:center;
+      padding-top:20px;
+      padding-left:40px;
+      margin-left:70px;
+      margin-right:70px;
+      
+    }
+    
+    .button {
+      border: none;
+      color: white;
+      padding: 10px 32px;
+      text-align: center;
+      text-decoration: none;
+      display: inline-block;
+      font-size: 13px;
+      margin: 1px 1px;
+      cursor: pointer;
+      border-radius: 3px;
+      background-color:black;
+    }
+    
+   
+  #render-wiki{
+    border: 1px solid black;
+    width:100%;
+    height:300px;
+    margin-bottom:60px;
+    margin-left:20px;
+    margin-right:60px;
+    margin-top:30px;
+    align-items:center;
+  }
+  
+  .container1{
+      height: 400px;
+      width: 600px;
+      border: 1px solid black; 
+      background-color:#f2f2f2;
+      display: flex;
+      flex-direction:column;
+      align-items:center;
+      padding-left:40px;
+      margin-left:70px;
+      margin-right:70px;
+      margin-top:20px;
+    }
+  .button1 {
+      border: none;
+      color: white;
+      padding: 11px 33px;
+      text-align: center;
+      text-decoration: none;
+      display: inline-block;
+      font-size: 13px;
+      cursor: pointer;    
+      background-color:black;
+      border-radius: 3px;
+      
+    }
+  
+  #render-dictionary{
+    border: 1px solid black;
+    width:400px;
```

Comment:
width is hard coded it should be 100% of the parent container and that should be relative not in pixels to be responsive.